app/routes/admin/reports/index.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  beforeModel() {
+    this._super(...arguments);
+    this.replaceWith('admin.reports.system-logs');
```

Comment:
@harshitagupta30 Why? According to the documentation, they are identical except replace with doesn't create history in the browser, transitionTo does create, so that there would be two entries for visiting the index route, one for the index route and then for the route it transitions to.